FAQ.md

```diff
@@ -124,10 +124,12 @@ That said, if you are _experiencing serious issues_ with your instance which you
 
 ## I changed my .env configuration but the error still appears/new config doesn't seem to be applied?
 
-After you edited your `.env` configuration file on a bare metal/VM setup, you always need to execute the dump-env command manually (this is **not** needed on Docker):
+After you edited your `.env` configuration file on a bare metal/VM setup, you always need to execute the `composer dump-env` command (in Docker you just restart the containers).
+
+Running the `post-upgrade` script will also execute `composer dump-env` for you:
 
 ```bash
-composer dump-env prod
+./bin/post-upgrade
```

Comment:
I'm not super knowledgeable on the command as a non-docker user but I assume there isn't a downside to running it when unneeded

Reply:
no there is no real down-side, maybe some minor cache clean-up, but in general there is no down side in running this command whenever you please.